package/requirements.txt

```
@@ -21,5 +21,6 @@ seaborn>=0.7.0,<=0.9
 sphinx==1.8.5
 sphinx_rtd_theme
 sphinx_sitemap
+sphinxcontrib.bibtex
```

```suggestion
sphinxcontrib-bibtex
```

The name of the module on PyPI is `sphinxcontrib-bibtex`